tests/test-cases/Group6-VIC-Machine/6-11-Debug.robot

```diff
@@ -35,3 +35,34 @@ Enable SSH and verify
 
     # delete the keys
     Remove Files  %{VCH-NAME}.key  %{VCH-NAME}.key.pub
+
+
+*** Test Cases ***
```

Comment:
only one test case table needed, not sure if it actually even supports two.  just eliminate this.